nvm.sh

```diff
@@ -3015,38 +3054,63 @@ nvm() {
       if [ -n "${NVM_LTS-}" ]; then
         VERSION="$(nvm_match_version "lts/${NVM_LTS:-*}")"
       elif [ -z "${PROVIDED_VERSION-}" ]; then
-        nvm_rc_version
+        # Get the .nvmrc version depending if quiet mode is on or not
+        if [[ $NVM_USE_SILENT -ne 1 ]]; then
+          nvm_rc_version
+        else
+          nvm_rc_version quiet
+        fi
         if [ -n "${NVM_RC_VERSION-}" ]; then
           PROVIDED_VERSION="${NVM_RC_VERSION}"
           VERSION="$(nvm_version "${PROVIDED_VERSION}")"
         fi
         unset NVM_RC_VERSION
         if [ -z "${VERSION}" ]; then
-          nvm_err 'Please see `nvm --help` or https://github.com/nvm-sh/nvm#nvmrc for more information.'
+          if [[ $NVM_USE_SILENT -ne 1 ]]; then
```

Comment:
```suggestion
          if [ $NVM_USE_SILENT -ne 1 ]; then
```
double bracket syntax is only supported in bash; nvm supports POSIX.